files/opt/hybris-tools/h-up

```diff
@@ -273,8 +268,22 @@ if __name__ == "__main__":
                 fout.writelines(data)
             
         if config["h_init"]:
-            ant_log_out.append(h_ant("initialize", config["username"], config["hybris_custom_ant_targets_path"]))
+            value_type = type(config["h_init"])
+            if value_type is bool:
+                ant_log_out.append(h_ant("initialize", config["username"], config["platform_home"]))
+            elif value_type is str:
+                ant_log_out.append(h_ant("initialize -DconfigFile=" + config["h_init"], config["username"],
```

Comment:
isinstance(value_type, str) should be used
ref: https://www.peterbe.com/plog/bool-is-int